src/Drivers/check_wfc.cpp

```diff
@@ -52,9 +52,7 @@ void print_help()
   cout << "      one of: J1, J2, J3.            default: J2"            << '\n';
   cout << "  -g  set the 3D tiling.             default: 1 1 1"         << '\n';
   cout << "  -h  print help and exit"                                   << '\n';
-  cout << "  -i  number of Monte Carlo steps.   default: 100"           << '\n';
   cout << "  -r  set the Rmax.                  default: 1.7"           << '\n';
-  cout << "  -s  set the random seed.           default: 11"            << '\n';
```

Comment:
This seed option should probably stay, and get hooked up.   It appears this code uses the same implicit seed all the time (unlike check_spo, which generates the seed from the time).